src/main/java/com/onarandombox/MultiverseCore/utils/MaterialConverter.java

```diff
@@ -32,11 +32,33 @@ public static Material convertConfigType(@NotNull ConfigurationSection config, @
      */
     @Nullable
     public static Material convertTypeString(@Nullable String value) {
```

Comment:
I'm pretty confused by the change to this method.

I can see that you've changed it to call `getLegacyType` when necessary, but you also added two more code blocks. What are they for?

Reply:
Its to support multiple formats, for example, if you want to get a granite material, you can pass in `1:1`, `granite` (on >1.13), or `stone:1` you will get the same material enum.